.gitignore

```diff
@@ -1,6 +1,7 @@
 # OSX
 #
 .DS_Store
+.env
```

Comment:
Why ignore the environment file ?

Reply:
there was and env file but being deleted before -> no need to ignore

Reply:
env file is not the file that should be delete or ignore. Please read more about .env file usage

Reply:
this env file is the one I added later before your last config. I will remove this ignore line